SPEC/FILES.md

```diff
@@ -709,11 +709,11 @@ ipfs.files.rm('/my/beautiful/directory', { recursive: true }, (err) => {
 
 #### `read`
 
-> Read a file.
+> Read a file into a [`Buffer`][b].
```

Comment:
@achingbrain what about the write methods?

Reply:
They were outside of the scope of the original conversation.  Currently you can pass a number of types of content to the `write` command and it'll convert them all into a pull-stream internally.  I've updated the docs in this PR to show what we now support.